Just_Dail_Scrapper.py

```diff
@@ -0,0 +1,68 @@
+from bs4 import BeautifulSoup
+import urllib
+import csv
+import constant
+
+
+def scraper():
+	def get_name(body):
+		return body.find('span', {'class':constant.nameclass}).a.string
```

Comment:
I am not sure if you are following 4 indents everywhere or they are varying. Please check